package.json

```diff
@@ -13,6 +13,7 @@
   },
   "scripts": {
     "ng": "ng",
+    "config": "ts-node set-env.ts",
     "postinstall": "webdriver-manager update --versions.chrome 2.40",
     "build": "ng build",
```

Comment:
do we nee to change this too?

Reply:
we will need to run config script wherever we use the environment as 'prod'